docs/03_improve_deploy_app/0302.md

```diff
@@ -121,16 +121,16 @@ If you are interested in learning more about Infrastructure as Code, there are m
 
 - The final Bicep script is [in the solution folder]((https://github.com/microsoft/TechExcel-Implementing-DevOps-practices-to-accelerate-developer-productivity/blob/main/Solution/Exercise-03/Task-2/main.bicep)).
 - The following az cli commands will create an application registration and then service principal. This uses a sample app registration name of `TechExcelUser`.
-  - `az ad app create --display-name TechExcelUser`. Copy the "appId" output to use in the next command, as well as to save this as a GitHub secret. Copy the "id" output to use when creating the federated credential.
-  - `az ad sp create --id {appId}`. Copy the "appId" to use in the next command for the {service principal ID}.
+  - `az ad app create --display-name TechExcelUser`. Copy the `appId` output to use in the next command. Save the `appId` as a GitHub secret called AZURE_CLIENT_ID. Copy the "id" output to use for the {app registration ID} when creating the federated credential.
+  - `az ad sp create --id {appId}`. Copy the "id" of this newly created service principal to use in the next command for the {service principal ID}. Save the `appOwnerOrganizationId` as a GitHub secret called `AZURE_TENANT_ID`
   - `az role assignment create --role contributor --scope /subscriptions/{subscription_id}/resourceGroups/{resource_group_name} --subscription {subscription_id} --assignee-object-id {service principal ID} --assignee-principal-type ServicePrincipal`.
   - Create a file called credentials.json and fill in the contents with your account name:
 
   ```json
     {
         "name": "GitHubDevOpsCredential",
         "issuer": "https://token.actions.githubusercontent.com",
-        "subject": "repo:{your_account}/TechExcel-Implementing-DevOps-practices-to-accelerate-developer-productivity-code:ref:refs/heads/main",
+        "subject": "repo:tibor19/TE-Implementing-DevOps:ref:refs/heads/main",
```

Comment:
Looks like this change shouldn't be here.

Reply:
You are right. It's not on my machine, but somehow appeared here.